src/templates/Brochure/BrochureTemplate.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FunctionComponent } from "react";
+import { BrochureTemplate } from "./BrochureTemplate";
+import { BrochureSample } from "./sample";
+
+export default {
+  title: "Brochure",
+  component: BrochureTemplate,
+  parameters: {
+    componentSubtitle: "Brochure template.",
+  },
+};
+
+export const BrochureEmpty: FunctionComponent = () => {
+  return <BrochureTemplate document={{} as any} handleObfuscation={() => {}} />; // when empty, visually should not show any dangling values
+};
```

Comment:
is there a need for empty in this marketing gig case?

Reply:
ok removed